index.html

```diff
@@ -109,6 +109,34 @@
     </script>
   </head>
   <body>
+    <!-- navbar -->
+    <div class="menu">
+      <nav id="navbar" class="navbar navbar-expand-lg navbar-light sticky-top">
+       <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+       <span class="navbar-toggler-icon"></span>
+      </button>
+       <div class="collapse navbar-collapse" id="navbarNav">
+         <ul class="navbar-nav text-center">
```

Comment:
i left a comment below abt this, but apparently its super ez to center these links by adding "mx-auto" here :''''')